tests/integrations/test_zk_evm.py

```diff
@@ -61,6 +61,24 @@ async def test_deploy(
         decimals = await erc_20.decimals()
         assert decimals == 18
 
+    async def test_deploy_erc721(
+        self,
+        deploy_solidity_contract: Callable,
+    ):
+        erc_721 = await deploy_solidity_contract(
+            "ERC721", "name", "symbol", caller_address=1
```

Comment:
are these arguments named? 
we could have a more visual setup:
```python
erc_721 = await deploy_solidity_contract(
            contract="ERC721", name="KakarotNft", symbol="KRTNFT", caller_address=1
)
```

Reply:
same for erc20

Reply:
yes they could be kwargs as well, they are directly given to the `web3.contract().constructor()`.
I can update this, but tbh I'd like to move ERC20 and ERC721 tests into their own files integrating all the tests from solmate repo, so I'm note sure it's worth doing anything here.

But it makes me realized that these utils aren't well documented, so in any case I'm adding some docstring, and consequently why not these args as kwargs